week1-2/views/FilterView.js

```diff
@@ -0,0 +1,57 @@
+import { storage } from "../Storage.js";
+import { on, qs } from "../util/domHelper.js";
+import { firstRender } from "./FirstRender.js";
+import View from "./View.js";
+
+const tag = "[FilterView]";
+export default class FilterView extends View {
+  constructor() {
+    console.log(tag, "[FilterView]");
+    super(qs("#filter_view"));
+    this.incomeElement = qs("[name=income]", this.element);
+    this.spendElement = qs("[name=spend]", this.element);
+    this.bindIncomeEvent();
+    this.bindSpendEvent();
+  }
+
+  bindIncomeEvent() {
+    on(this.incomeElement, "change", () => this.handleIncomeClick());
+  }
+
+  bindSpendEvent() {
+    on(this.spendElement, "change", () => this.handleSpendClick());
+  }
+
+  handleIncomeClick() {
+    this.updateView();
+  }
+
+  handleSpendClick() {
+    this.updateView();
+  }
```

Comment:
이 두 함수가 결국엔 똑같은 함수를 가리키고 있는거니까 이런 식으로 하나로 묶어줘도 될 것 같다 !!?
```suggestion
  handleClickFilteredBtn() {
    this.updateView();
  }

```

Reply:
엇... 놓치고 있었다 ㅋㅋㅋ 세심한 피드백 고마워!!